modules/virt-listing-vmis-cli.adoc

```diff
@@ -0,0 +1,17 @@
+
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-manage-vmis.adoc
+
+[id="virt-listing-vmis-cli_{context}"]
+= Listing all virtual machine instances using the CLI
+
+You can list both virtual machine instances (VMIs) owned by virtual machines and standalone (VMIs) in your cluster by using the `oc` command-line interface (CLI).
```

Comment:
I would rephrase this like:

"You can list all virtual machine instances (VMIs) in your cluster, including standalone VMIs and those owned by virtual machines, by using the `oc` command-line interface (CLI)."

Reply:
Updated.